src/components/ComponentSettingsModal.jsx

```diff
@@ -87,9 +88,25 @@ const ComponentSettingsModal = ({
         fileUrl: setFileUrl,
     }
 
+    useQuery(
+      [RESOURCE_ROOM_NAME_KEY, siteName],
+      () => getResourceRoomName(siteName),
+      {
+        retry: false,
+        onSuccess: (retrievedName) => {
+          setResourceRoomName(retrievedName)
```

Comment:
nit, what do you think of unpacking the data from the useQuery, as in:
`const { data: resourceRoomName } = useQuery(...)` so that we don't need a state hook?

Reply:
Great suggestion, I've made the change in https://github.com/isomerpages/isomercms-frontend/pull/478/commits/c008d3f9be261d6048df85984c322193c7676767